files/image_config/caclmgrd/caclmgrd

```diff
@@ -84,6 +84,14 @@ class ControlPlaneAclManager(object):
         "SSH": {
             "ip_protocols": ["tcp"],
             "dst_ports": ["22"]
+        },
+        "IP_ANY": {
+            "ip_protocols": ["any"],
+            "dst_ports": ["0"]
+        },
+        "IPV6_ANY": {
+            "ip_protocols": ["any"],
+            "dst_ports": ["0"]
```

Comment:
Do we really need `IPV6_ANY`? The service here is not specific to v4 or v6. Both v4 and v6 rules could be configured with the service as `IP_ANY`. Maybe we could even change it to simply `ANY`.

Reply:
Sure Joe, I'll just make "ANY": {"ip_protocols": ["any"], "dst_ports": ["0"]} change and verify again.

Reply:
What happened to this change? Is there a reason you kept both `"IP_ANY"` and `"IPV6_ANY"`?

Reply:
Will push the changes soon, confused because the PR is in approved state already.